### PR TITLE
Add some information and break loop when a task failed incomplete

### PR DESCRIPTION
**Pull request by bzheng888**
### What changes are proposed in this pull request?
Add more information and break the taskId check loop when a task failed.   

### Why are the changes needed?
Not enough information for us to know which task failed , no need to check other taskId in the loop, just break.  
Please clarify why the changes are needed. For instance,
Get more information when task failed, skip useless loop.

### Does this PR introduce any user facing changes?
No
Please list the user-facing changes introduced by your change, including
No

### CODE REVIEW

**Review by maobaolong — APPROVED** [review]
It lgtm, thanks for this improvement.

File: job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java [hunk 1]
```diff
@@ -224,8 +224,10 @@ public void failTasksForWorker(long workerId) {
         }
         taskInfo.setStatus(Status.FAILED);
         taskInfo.setErrorType("JobWorkerLost");
-        taskInfo.setErrorMessage("Job worker was lost before the task could complete");
+        taskInfo.setErrorMessage(String.format("Job worker(%s) was lost before"
+                + "the task(%d) could complete", taskInfo.getWorkerHost(), taskId));
```
Comment (apc999) [inline job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java:228]:
```suggestion
        taskInfo.setErrorMessage(String.format("Job worker(%s) was lost before "
                + "the task(%d) could complete", taskInfo.getWorkerHost(), taskId));
```

**Review by yuzhu — APPROVED** [review]
LGTM , thanks

**Review by apc999 — APPROVED** [review]

**Comment by bzheng888** [comment]
@apc999 @yuzhu PTAL, this pr seems ready to merge.

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**